.nvmrc

```diff
@@ -0,0 +1 @@
+v16.3.0
```

Comment:
I think we should make this 14.9.0. AWS doesn't support node 16 yet in serverless functions